src/Command/Ssh/SshKeyCreateCommand.php

```diff
@@ -88,12 +88,12 @@ protected function determineFilename(InputInterface $input, OutputInterface $out
       $this->validateFilename($filename);
     }
     else {
-      $question = new Question('<question>Please enter a filename for your new local SSH key:</question> ');
+      $question = new Question('<question>Please enter a filename for your new local SSH key:</question> ', 'id_rsa_acquia');
```

Comment:
Added default ssh key name.

Reply:
Do we have a check to ensure to not overwrite an existing key with the same name?

Reply:
Should we also add a sequence number of the IDE - in case of multiple entitlements. THey could have different SSH keys

Reply:
No, you'd never have keys for multiple IDEs inside of one IDE. The label stored in Cloud does have such a suffix, but it's not required for a local filename.